packages/vest/testUtils/resetState/index.js

```diff
@@ -1,19 +1,19 @@
 import { OPERATION_MODE_STATEFUL } from '../../src/constants';
 import * as state from '../../src/core/state';
-import getSuiteState from '../../src/core/state/getSuiteState';
-import runRegisterSuite from '../runRegisterSuite';
+import getState from '../../src/core/suite/getState';
+import runRegister from '../runRegister';
 
-const resetState = (suiteId, suiteName = suiteId) => {
+const resetState = (Id, Name = Id) => {
   state.set(() => null);
   state.register();
 
-  if (suiteName) {
-    runRegisterSuite({
-      suiteId,
-      name: suiteName,
+  if (Name) {
+    runRegister({
+      Id,
+      name: Name,
       operationMode: OPERATION_MODE_STATEFUL,
     });
-    return getSuiteState(suiteId);
+    return getState(Id);
```

Comment:
Looks like the tests are failing due to this change. The runRegister function ends up here: https://github.com/ealush/vest/blob/4f246971f7b280e2bf02b3b0d2fe45f56547affd/packages/vest/src/core/state/index.js#L27

This function expects suiteId and not `Id`

You can fix nearly all of the failed tests by reverting these names back (suiteId, suiteName). I am assuming this was just a bad copy/paste - I see you even kept the capitalized I and N.